### PR TITLE
fix: Remove whitespace from glue job sec configuration Id

### LINKED ISSUE #274: ts: GlueTransform stage failed in step functions
### Describe the bug

GlueTransform stage failed in step functions

(line 81 and 93 is correct?) i believe its causing the glue transform stage to fail: https://github.com/awslabs/aws-ddk/blob/typescript-conversion/src/stages/glue-transform.ts

Error: 

`"ErrorMessage": "JobName:gluetransformstagegluetrans-e1giOK9FBkBm and JobRunId:jr_5a943c9e0b6bfc6f9dec9c8f5db1c4b15c105bc14521e82de08aa50259e111ac failed to execute with exception 2 validation errors detected: Value '/aws-glue/jobs/glue-transform-stage Job-security-config-role/FileStandardizationPipeli-gluetransformstagegluetr-DCMJREXB2DW1/error' at 'logProperties.stdErrLogGroupName' failed to satisfy constraint: Member must satisfy regular expression pattern: [\\.\\-_/#A-Za-z0-9]+; Value '/aws-glue/jobs/glue-transform-stage Job-security-config-role/FileStandardizationPipeli-gluetransformstagegluetr-DCMJREXB2DW1/output' at 'logProperties.stdOutLogGroupName' failed to satisfy constraint: Member must satisfy regular expression pattern: [\\.\\-_/#A-Za-z0-9]+ (Service: AWSGlueJobExecutor; Status Code: 400; Error Code: InvalidInputException; Request ID: a1c9224f-8918-4082-876c-dae45e0612a1; Proxy: null)",`

### How to Reproduce

Deploy an ddk example with gluetransform. 

- cdk synth should work
- cdk deploy should work
- end to end test failed with error 

### Expected behavior

Synth, Deploy and end to end test should be successfuly

### Your project

_No response_

### Screenshots

_No response_

### Environment

```
Provide your `pip list` output, particularly the version of the AWS DDK library you used. Providing this information may significantly improve resolution times.
```

### OS

Mac

### Python version

3.9

### AWS DDK version

1.0.0b

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by malachi-constant**
### Feature or Bugfix
- Remove whitespace from sec configuration Id

### Relates 
- Closes #274 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: DDKIntegrationTestsD8472609-5SCHWjldmWcf
* Commit ID: 2e34674a7df5a3a3e6ea0d64c1644e942ea5fe7a
* Result: SUCCEEDED
* [Build Logs](https://9hm1y4sor9.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=19d213d2-fac9-4c17-a514-cf1327f7784a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/core/glue-defaults.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export class GlueDefaults {
       maxRetries: 1,
       timeout: cdk.Duration.hours(10),
       securityConfiguration: new glue.SecurityConfiguration(scope, id, {
-        securityConfigurationName: `${id}-security-config`,
```
Comment (LeonLuttenberger) [inline src/core/glue-defaults.ts:13]: Do we need to provide a name for the security configuration?

Comment (malachi-constant) [reply src/core/glue-defaults.ts:13]: I don’t recall why I needed the name in there. I’ll validate generation works with our integration test

Comment (malachi-constant) [reply src/core/glue-defaults.ts:13]: Yea this unnecessary, taking out

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: DDKIntegrationTestsD8472609-5SCHWjldmWcf
* Commit ID: c21134a41bdff2b05f2ef05bc487a0c6dbe2c240
* Result: SUCCEEDED
* [Build Logs](https://9hm1y4sor9.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=d983266d-50d4-4bb4-b612-f5261c491d0d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by LeonLuttenberger — APPROVED** [review]

**Review by anmolsgandhi — APPROVED** [review]

**Pull request merged**